cmd/placement/README.md

```diff
@@ -33,24 +33,33 @@ go run cmd/placement/main.go --kcp-kubeconfig=<path to kcp admin.kubeconfig> -v
 #### A short demo
 The short demo shows the Placement State Machine described in this [document](https://docs.google.com/document/d/1AzyjuyjNIDVAXEGHslaggltIQ9Cs8pLCzc9Ma_RBmuM/edit#heading=h.vmt32rdidje6), or in this [code block](https://github.com/kcp-dev/kcp/blob/fb4d4a42373ea4da001b8c88396eabaf6f825be1/pkg/apis/scheduling/v1alpha1/types_placement.go#L123-L134).
 
-In a user's workspace, say `root:my-org:dev`, create Placement `dev`:
+In the `root:compute` workspace, create Location `foo`:
 ```console
-kubectl create -f config/samples/placement_dev.yaml
+kubectl create -f config/samples/location_foo.yaml
 ```
-The status of Placement `dev` should show `phase: Pending`.
 
-Switch to the `root:compute` workspace, create Location `foo`:
+Switch to a user's workspace, say `root:my-org:dev`, create Placement `dev`:
```

Comment:
That's not what a user's workspace looks like in the kcp version used here.  I see the following.

```
(base) mspreitz@mjs12 edge-mc % kubectl ws \~
Current workspace is "kvdk2spgmbix".
```

Reply:
Improving this in https://github.com/kcp-dev/edge-mc/pull/138, thanks! @MikeSpreitzer